Source/WebCore/platform/mediastream/RealtimeMediaSource.cpp

```diff
@@ -402,7 +402,7 @@ bool RealtimeMediaSource::supportsSizeAndFrameRate(std::optional<IntConstraint>
     }
 
     // Each of the non-null values is supported individually, see if they all can be applied at the same time.
-    if (!supportsSizeAndFrameRate(WTFMove(width), WTFMove(height), WTFMove(frameRate))) {
+    if (!supportsSizeAndFrameRate(width, height, WTFMove(frameRate))) {
```

Comment:
Given width, height and frameRate are std::optional<double or int>, we should probably not move them at all in this method.